src/seedu/addressbook/ui/TextUi.java

```diff
@@ -124,46 +124,10 @@ public void showToUser(String... message) {
     public void showResultToUser(CommandResult result) {
         final Optional<List<? extends ReadOnlyPerson>> resultPersons = result.getRelevantPersons();
         if(resultPersons.isPresent()) {
-            showPersonListView(resultPersons.get());
+        	Formatter format = new Formatter(resultPersons.get());
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.